packages/flutter/lib/src/foundation/diagnostics.dart

```diff
@@ -115,8 +136,38 @@ enum DiagnosticsTreeStyle {
   ///  * [SliverGeometry], which uses this style.
   whitespace,
 
+  /// Render the tree without indenting children at all.
+  ///
+  /// See also:
+  ///
+  ///  * [DiagnosticsStackTrace], which uses this style.
+  flat,
+
   /// Render the tree on a single line without showing children.
   singleLine,
+
+  /// Render the tree on a single line without showing children.
+  ///
+  /// See also:
+  ///  * [FlutterErrorDetails], which uses this style to display the header
+  ///    describing the context where an error occurred.
+  headerLine,
+
+  /// Render the tree on a single line with the name and value on separate
```

Comment:
This seems oxymoronic. How can it be a single line if name and value are presented on separate lines, which makes it at least two lines?

Reply:
yes that is a bad name.
some alternate ideas:
* whitespaceNoChildren <-- that is what it is
* errorProperty <-- that is what it is for.

Reply:
Fixed this. I've renamed this to errorProperty as that is what it is.